<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxFruitNonExclusive">
       <property name="title">
        <string>Choose(NonExclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxBanana">
          <property name="text">
           <string>Banana</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxApple">
          <property name="text">
           <string>Apple</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxLemon">
          <property name="text">
           <string>Lemon</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxKgExclusive">
       <property name="title">
        <string>Choose(Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox1">
          <property name="text">
           <string>1 Kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox2">
          <property name="text">
           <string>2 KG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox3">
          <property name="text">
           <string>3 Kg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonBuy">
     <property name="text">
      <string>Buy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
